server/channels/app/server.go

```diff
@@ -1825,14 +1825,14 @@ func runDNDStatusExpireJob(a *App) {
 func runPostReminderJob(a *App) {
 	if a.IsLeader() {
 		withMut(&a.ch.postReminderMut, func() {
```

Comment:
Can all methods in `runPostReminderJob` share a `request.CTX`?

Reply:
I assume `request.CTX` will be different if cluster leader changes?

Reply:
No, it shouldn't be different.